src/js/components/devices/expanded-device.js

```diff
@@ -37,8 +41,41 @@ import MonitorDetailsDialog from './device-details/monitordetailsdialog';
 import DeviceNotifications from './device-details/notifications';
 import DeviceTwin from './device-details/devicetwin';
 
+const useStyles = makeStyles()(theme => ({
+  gatewayChip: {
+    color: theme.palette.grey[900],
+    path: {
+      fill: theme.palette.grey[900]
+    }
+  },
+  gatewayIcon: {
+    width: 20
+  }
+}));
+
 const refreshDeviceLength = 10000;
 
+const GatewayNotification = ({ device, docsVersion }) => {
+  const ipAddress = getDemoDeviceAddress([device]);
+  const { classes } = useStyles();
+  return (
+    <MenderTooltipClickable
+      placement="bottom"
+      title={
+        <div style={{ maxWidth: 350 }}>
+          For information about connecting other devices to this gateway, please refer to the{' '}
+          <a href={`https://docs.mender.io/${docsVersion}server-integration/mender-gateway`} target="_blank" rel="noopener noreferrer">
+            Mender Gateway documentation
+          </a>
+          . The device is reachable via <i>{ipAddress}</i>.
```

Comment:
As above we are referring to "other devices", I would be here more explicit with
```suggestion
          . This device is reachable via <i>{ipAddress}</i>.
```

or even:
```suggestion
          . The gateway device is reachable via <i>{ipAddress}</i>.
```

Reply:
Thanks, I chose the "this" option, as the gateway option seemed less specific to the expanded device, while being too specific to the gateway functionality (since the device is likely also reachable for non-gateway functionalities on that IP)